packages/ui-extensions/docs/surfaces/checkout/staticPages/upgrading-to-2025-10.doc.ts

```diff
@@ -83,11 +83,9 @@ As of \`2025-10\`, Shopify recommends Preact for UI extensions. Update the depen
     {
       type: 'GenericAccordion',
       anchorLink: 'update-typescript-configuration',
-      title: 'Update TypeScript Configuration',
+      title: 'TypeScript Configuration',
       sectionContent: `
-**Skip this step if not using TypeScript.**
-
-Update your extension config at a path like \`extensions/{extension-name}/tsconfig.json\`. You do **not** need to change your app's root \`tsconfig.json\` file.
+Get full IntelliSense and auto-complete support by adding a config file for your extension at \`extensions/{extension-name}/tsconfig.json\`. You do **not** need to change your app's root \`tsconfig.json\` file.
```

Comment:
should we say here something like "even if you're not using TypeScript"?

Reply:
happy to further improve next week 😄